src/options.js

```diff
@@ -2,6 +2,41 @@ import merge from 'deepmerge';
 import Handlebars from 'handlebars';
 import yaml from 'js-yaml';
 import marked from 'marked';
+import frontMatter from 'front-matter';
+
+var parsers = {
+  content: {
+    pattern: /\.(html|hbs|handlebars)$/,
```

Comment:
I noticed this is the only one with a regular expression literal instead of a string.